view/rooms/app.js

```diff
@@ -622,16 +623,17 @@ function userEnterRoom() {
   const message = currentUser.fullName + ' has joined the room at ' +
       moment().format('YYYY-MM-DD HH:mm Z');
   const eventType =  {
-      type: 'User',
-      user: currentUser,
-      isActive: true,
-    };
+    type: 'User',
+    user: currentUser,
+    isActive: true,
+  };
   const events = {
-      log: message,
-      context: eventType,
-      userId: _user.id,
-      roomId: parseInt(ROOM_ID, 10),
-    };
+    log: message,
+    actionType: eventType.type + 'Login',
```

Comment:
Just a minor thing but should this be named Join/Leave instead of Login/Logout, just a preference thing though